germanium/storage.py

```diff
@@ -0,0 +1,210 @@
+import os
```

Comment:
most of it is copy of dj-inmemorystorage (https://pypi.org/project/dj-inmemorystorage/) But I must fix bugs a improve it for testing purposes